extide/gradle/netbeans-gradle-tooling/src/main/groovy/org/netbeans/modules/gradle/tooling/NbProjectInfoBuilder.groovy

```diff
@@ -258,8 +258,10 @@ class NbProjectInfoBuilder {
                             model.info["sourceset_${sourceSet.name}_classpath_annotation"] = storeSet(sourceSet.compileClasspath.files)
                         }
                     }
-                    model.info["sourceset_${sourceSet.name}_configuration_compile"] = sourceSet.compileConfigurationName;
-                    model.info["sourceset_${sourceSet.name}_configuration_runtime"] = sourceSet.runtimeConfigurationName;
+                    beforeGradle('7.0') {
+                        model.info["sourceset_${sourceSet.name}_configuration_compile"] = sourceSet.compileConfigurationName;
```

Comment:
I've tried to find where this information is used in gradle project, but I've found no references to `GradleJavaSourceSet.getRuntimeConfigurationName()` ?
Is this information superseded by something or maybe should be the `getRuntimeConfigurationName()` annotated as `@deprecated` as they won't provide useful info for Gradle >= 7?